db/schema.sql

```diff
@@ -413,3 +413,13 @@ CREATE TABLE `report_sentry_issues` (
 ) ENGINE=InnoDB AUTO_INCREMENT=8461 DEFAULT CHARSET=utf8mb3;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+DROP TABLE IF EXISTS `report_sentry_crash_free_rate`;
```

Comment:
Note: If we want to integrate the changes for `production-daily.yml` we need to run this on `production` database.